lightning/src/ln/channel.rs

```diff
@@ -2915,6 +2934,10 @@ where
 	/// Similar to [`Self::signer_pending_commitment_update`] but we're waiting to send a
 	/// [`msgs::ChannelReady`].
 	signer_pending_channel_ready: bool,
+	// Upon receiving a [`msgs::ChannelReestablish`] message with a `next_remote_commitment_number`
+	// indicating that our state may be stale, we set this to the received last-revoked commitment
+	// number and secret.
```

Comment:
I think you deleted some useful context here on the last push, regarding that we cache this info until the signer is ready to verify that our state is actually stale